tex/chapters/Results.tex

```diff
@@ -2,12 +2,221 @@
 
 \chapter{Results}\label{cha:results}
 
-\section{Introduction}
+In this chapter, the implementation of a fingerprint function within the Axelrod-Python library will be examined.
+This includes the addition of two strategy transformers, the Dual and JossAnn as defined in \ref{}. %TODO
+Then several results will be presented where analytical fingerprints are compared with analytical ones.
+A discussion that compares different fingerprints of strategies within Axelrod-Python will also be given.
 
+\section{The Dual}
```

Comment:
A slighter better introduction needed. This section should be "Numerical computation of the Dual". However I actually think it could be moved to a later section about using "Using probes that are not natural FSM strategies". You should also add that that section will be in this chapter and that's a novel result and has not been done before.